client/notices.go

```diff
@@ -96,6 +107,7 @@ type Notice struct {
 	ID            string            `json:"id"`
 	Type          NoticeType        `json:"type"`
 	Key           string            `json:"key"`
+	UserID        *int              `json:"user-id"`
```

Comment:
Let's make this `omitempty` too for consistency:

```suggestion
	UserID        *int              `json:"user-id,omitempty"`
```